Makefile

```diff
@@ -2,6 +2,7 @@
 # Run "make clean" to remove automatically generated files
 ruff:
 	python -m ruff --diff src/wikitextprocessor
+	python -m ruff --diff tests
```

Comment:
I'd recommend `ruff format .`, this command will format the files for you.

Reply:
I'm formatting by hand inside the editor (Helix, `:format` command), because I had trouble with venv breaking ruff installs when I was in the wrong environment and basically destroying files when using auto-format on save. Just being careful. The command should probably be just named "make ruffcheck" or something...